Cargo.toml

```diff
@@ -33,6 +35,10 @@ zerocopy = ["dep:zerocopy"]
 # Derives the `Serialize`, `Deserialize`, and `Archive` traits from the [`rkyv`](https://crates.io/crates/rkyv) crate for the `Primes` struct.
 rkyv = ["dep:rkyv"]
 
+# Calls the Machine-Prime library for much faster primality testing
+fastprime = ["dep:machine-prime"]
```

Comment:
Could the name of the feature be changed to `fast_test` since it is specifically testing it speeds up?